sdk_files/OneSignalSDKWorker.js

```diff
@@ -1 +1 @@
-importScripts('https://cdn.onesignal.com/sdks/OneSignalSDKWorker.js');
+importScripts("https://cdn.onesignal.com/sdks/web/v16/OneSignalSDK.sw.js");
```

Comment:
It's a little curious that we have chosen to version our hosted Service Worker. Is there any documentation about why we decided to do this?

Reply:
We need to support both the player model SDK as well as the new user model SDK which use different workers